wallet/wallet-cli-lib/src/repl/mod.rs

```diff
@@ -57,7 +61,20 @@ pub fn get_repl_command(cold_wallet: bool) -> Command {
 
     // Customize the help template for all commands to make it more REPL friendly
     for subcommand in repl_command.get_subcommands_mut() {
-        *subcommand = subcommand.clone().help_template(COMMAND_HELP_TEMPLATE);
+        if let Some(desc) =
+            COLD_WALLET_DESC.methods.iter().chain(WALLET_DESC.methods).find_map(|method| {
+                method
+                    .name
+                    .split('_')
+                    .zip(subcommand.get_name().split('-'))
+                    .all(|(x, y)| x == y)
+                    .then_some(method.description)
+            })
+        {
+            *subcommand = subcommand.clone().help_template(COMMAND_HELP_TEMPLATE).about(desc);
+        } else {
+            *subcommand = subcommand.clone().help_template(COMMAND_HELP_TEMPLATE);
+        }
```

Comment:
This is great! I think the only concern is that the parameters have the same name. This can be done later using `syn` by parsing both interfaces. But I guess this is good enough for now. Not sure whether we should create an issue for this.